pkg/query/table.go

```diff
@@ -262,46 +267,33 @@ func newTableBuilder(mem memory.Allocator) *tableBuilder {
 	return tb
 }
 
-func (tb *tableBuilder) populateCallerAndCalleeData() {
-	reverseFunctionsMap := map[int]string{}
-	for k, v := range tb.functions {
-		reverseFunctionsMap[v] = k
+func getKeys(m map[int]bool) []int64 {
```

Comment:
since Go 1.20 or something there's the `maps` package that has a generic `Keys` function, so yu can just use `maps.Keys(...)`, no need to define these anymore per type